src/query/graphite/native/aggregation_functions.go

```diff
@@ -22,6 +22,7 @@ package native
 
 import (
 	"fmt"
+	"github.com/m3db/m3/src/query/block"
```

Comment:
Move this to group of imports with the other "github.com/m3db/m3"... prefix imports.